SonarQube.Client.Tests/SonarQubeService_TestBase.cs

```diff
@@ -106,7 +106,12 @@ protected async Task ConnectToSonarQube(string version = "5.6.0.0", string serve
         protected void ResetService()
         {
             messageHandler.Reset();
-            service = new SonarQubeService(messageHandler.Object, UserAgent, logger, requestFactorySelector, secondaryIssueHashUpdater.Object);
+            service = CreateTestSubject();
+        }
+
+        protected internal virtual SonarQubeService CreateTestSubject()
```

Comment:
I can't think of anything nicer with less changes :/

Reply:
Me neither.